src/main/java/com/SpringBoot/springbootRef/Controller.java

```diff
@@ -0,0 +1,73 @@
+package com.SpringBoot.springbootRef;
+
+import com.SpringBoot.springbootRef.Registration.Register;
+import com.SpringBoot.springbootRef.Registration.RegisterRequest;
+import com.SpringBoot.springbootRef.Registration.RegisterService;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@CrossOrigin
+public class Controller {
+    private final RegisterService registerService;
+
+    public Controller(RegisterService registerService) {
+        this.registerService = registerService;
+    }
+    @GetMapping("api/v1/registers")
+    public List<Register> GetRegisters(){
+        return registerService.GetRegisters();
+    }
+
+    @GetMapping("api/v1/registers/{register_Id}")
+    public Optional<Register> GetRegister(@PathVariable("register_Id") Integer Id){
+        return registerService.GetRegister(Id);
+    }
+    @PostMapping("api/v1/registers")
+    public ResponseEntity<String> saveRegister(@RequestBody RegisterRequest registerrequest) throws Exception {
```

Comment:
using camelCase consistently for method names, such as getRegisters, getRegister, and saveRegister

Reply:
Changes Updated. Kindly review it.